BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Views/SkontoWithoutDiscountPriceView.swift

```diff
@@ -32,7 +32,6 @@ class SkontoWithoutDiscountPriceView: UIView {
 
     private func setupView() {
         translatesAutoresizingMaskIntoConstraints = false
-        backgroundColor = .giniColorScheme().bg.inputUnfocused.uiColor()
```

Comment:
[clarification] Is this ok that we removed this?

Reply:
yes, it was extra and this background is not needed